ruby/rust-toolchain

```diff
@@ -0,0 +1,6 @@
+# This file specifies the Rust version used to develop and test the Ruby
```

Comment:
Shall we name the file `rust-toolchain.toml` . Both `rust-toolchain` and `rust-toolchain.toml` are valid, but I think with extension is preferable .

Reply:
Good point. I've pushed a commit to test that in CI, and if all is well then :+1: